tests/test_catmath.py

```diff
@@ -1,19 +1,39 @@
 from catinabox import catmath
+import pytest
 
 
-def test__cat_years_to_hooman_years__middle_age__succeeds():
-    assert True
+@pytest.mark.parametrize(('tc', 'expct'), [
+    (6, 30),
+    (5, 25),
+    (4, 20),
+    (10, 50)
```

Comment:
In retrospect I don't think my values align with the function name, they should have been in the 30-50 year range(middle age).